modules/swagger-codegen/src/main/resources/typescript-angular2/variables.mustache

```diff
@@ -1,9 +1,9 @@
 import { {{{injectionToken}}} } from '@angular/core';
 
-export const BASE_PATH = new {{{injectionToken}}}('basePath');
+export const BASE_PATH = new {{{injectionToken}}}<String>('basePath');
```

Comment:
Please see: https://angular.io/docs/ts/latest/api/core/index/InjectionToken-class.html
I changed it because my app just wouldn't compile otherwise.